### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by SiTingST**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -44,36 +44,28 @@ Vax@NUS is a **one stop management app to efficiently track and schedule COVID-1
...
-* Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
-
-* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
 
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
   
+* For commands with prefixs,
```
Comment (yienyoong) [inline docs/UserGuide.md]: This is a typo, it should be "Prefixes" :)

Comment (SiTingST) [reply docs/UserGuide.md]: Ohhhh, yes, thanks for pointing it out!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -44,36 +44,28 @@ Vax@NUS is a **one stop management app to efficiently track and schedule COVID-1
...
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
-
-* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
 
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
   
+* For commands with prefixes, 
+
```
Comment (dannn-tran) [inline docs/UserGuide.md]: Should this be "For commands with prefixed parameters" or "For parameters with a prefix"?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -44,36 +44,28 @@ Vax@NUS is a **one stop management app to efficiently track and schedule COVID-1
...
 
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
   
+* For commands with prefixes, 
+  
+   * Parameters can be in any order.<br>
+   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+  
+   * If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
```
Comment (dannn-tran) [inline docs/UserGuide.md]: Should it be "but you specify" instead of "but you specified"?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -406,6 +371,34 @@ If unindicated, the `School Residence` field will default to `DOES NOT LIVE ON C
...
+<br>
+**A**: Unfortunately, the deletion is permanent. You will have to manually re-add the student.
+
+**Q**: What can be included in medical details?
+<br>
+**A**: Allergies and past medical history.
+
+**Q** : Will I be able to add other types of appointments besides vaccination appointments?
+<br>
+**A** : No, the current version only allows you to add vaccination appointments and not any other type of appointments.
```
Comment (dannn-tran) [inline docs/UserGuide.md]: @SiTingST in the addAppt feature section, I wrote that vaccinated students can also have an appointment scheduled for follow-ups. Should I edit my part to say "booster dose" or should the Q&A here say that currently it doesn't differentiate different types of appointments?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -406,6 +371,34 @@ If unindicated, the `School Residence` field will default to `DOES NOT LIVE ON C
...
+<br>
+**A**: Allergies and past medical history.
+
+**Q** : Will I be able to add other types of appointments besides vaccination appointments?
+<br>
+**A** : No, the current version only allows you to add vaccination appointments and not any other type of appointments. 
+
+**Q** : Am I able to reassign an appointment to another student?
+<br>
+**A** : No, you will need to add a new appointment for the new student.
```
Comment (dannn-tran) [inline docs/UserGuide.md:389]: Should the ans say "No, you will need to delete the appointment and add a new one for the new student" instead?

File: docs/UserGuide.md [hunk 6]
```diff
@@ -406,6 +371,34 @@ If unindicated, the `School Residence` field will default to `DOES NOT LIVE ON C
...
+<br>
+**A** : No, you will need to add a new appointment for the new student. 
+
+**Q**: Why are vaccinated students able to receive new appointments?
+<br>
+**A**: Beyond the two shots currently required for the approved Pfizer-BioNTech and Moderna vaccines in Singapore, booster shots could be required in the future to prolong immunity and provide protection against fast-emerging COVID-19 variants. 
+
+**Q**: How long is a vaccination appointment?
+<br>
+**A**: The vaccination appointment lasts 30 mins, which includes registration, administering the vaccine and observation.
```
Comment (dannn-tran) [inline docs/UserGuide.md]: Should this be "A vaccination appt" or "Each..." or "Every" instead of "The"?

File: docs/UserGuide.md (hunk 2 above)
Comment (SiTingST) [reply docs/UserGuide.md]: "For commands with prefixed parameters" sounds clearer!

File: docs/UserGuide.md (hunk 3 above)
Comment (SiTingST) [reply docs/UserGuide.md]: Yup!

File: docs/UserGuide.md (hunk 4 above)
Comment (SiTingST) [reply docs/UserGuide.md]: I think its less confusing if we just remove this question?

File: docs/UserGuide.md (hunk 6 above)
Comment (SiTingST) [reply docs/UserGuide.md]: Yup, each is better. Thanks!

File: docs/UserGuide.md (hunk 5 above)
Comment (SiTingST) [reply docs/UserGuide.md:389]: I think the current version is okay?

Comment (dannn-tran) [reply docs/UserGuide.md:389]: okie!

File: docs/UserGuide.md [hunk 7]
```diff
@@ -425,8 +406,8 @@ Action | Format, Examples
 **Delete Student** | `delete MATRICULATION_NUMBER` e.g., `delete A1234567X`
 **Filter Students** | `filter VACCINATION_STATUS`  e.g., `filter vaccinated`, `filter unvaccinated`, <br> `filter FACULTY ` e.g., `filter COM` <br> `filter SCHOOL_RESIDENCE` e.g., `filter RVRC` 
 **View Student Statistics** | `stats FACULTY` e.g., `stats COM` <br> `stats SCHOOL_RESIDENCE` e.g., `stats RC4` <br> `stats NUS` <br> `stats all` 
-**Add Appointment** | `addAppt MATRICULATION_NUMBER d/DATE ts/START_TIME` <br> e.g., `addAppt A1234567X d/2021-12-13 ts/13:00`
-**Edit Appointment** | `editAppt MATRICULATION_NUMBER d/DATE_YYYY-MM-DD ts/START_TIME_HH:MM` <br> e.g.,` editAppt A1234567X d/2021-12-13 ts/14:00`
+**Add  Appointment** | `addAppt i/MATRICULATION_NUMBER d/DATE ts/START_TIME` <br> e.g., `addAppt i/A1234567X d/2021-12-13 ts/13:00`
```
Comment (dannn-tran) [inline docs/UserGuide.md]: Should `i/` be removed? and is there an extra space between "Add" and "Appointment"?

Comment (SiTingST) [reply docs/UserGuide.md]: Yup, it should be! Thanks for pointing it out!

**Review by dannn-tran — APPROVED** [review]
LGTM!

**Pull request merged**